components/email/src/Email.svelte

```diff
@@ -1111,21 +1083,54 @@
           }
         }
       }
-      .subject-snippet-date {
-        .desktop-subject-snippet {
-          display: none;
+      .subject-snippet-attachment {
+        padding: $spacing-xs;
+        padding-top: 0.4rem;
+        overflow: hidden;
+        .subject-snippet {
+          display: flex;
+          flex-direction: column;
+          gap: 0.5rem;
+          color: var(--nylas-email-snippet-color, var(--grey-dark));
+
+          .subject {
+            color: var(--nylas-email-subject-color, var(--black));
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          }
+
+          .snippet {
+            overflow: hidden;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+          }
         }
+
         .attachment {
-          &.desktop {
-            display: none;
+          margin-top: $spacing-xs;
+          gap: 1rem;
+          display: flex;
+          overflow-x: scroll;
```

Comment:
Could we make this `auto` ? Notice the second email here: 
![image](https://user-images.githubusercontent.com/713991/146558443-6b0de72c-717c-4db2-a944-a3c7d5700759.png)